QRCoder/QRCodeGenerator.cs

```diff
@@ -1043,6 +1023,8 @@ private static byte[] BinaryStringToBitBlockByteList(BitArray bitString, int off
             }
 
             return blocklist;
+
+            void ThrowArgumentException(string paramName) => throw new ArgumentOutOfRangeException(paramName, "Count must be a multiple of 8.");
```

Comment:
This helper is used only in once place, so I would remove the parameter and hard-code it here.
That avoid the load of the constant string at the call-site, thus makes the happy-path smaller.

The method could then be named `ThrowCountNotMultipleOf8` to reflect that change.